thoth/user_api/api_v1.py

```diff
@@ -104,6 +108,34 @@
 _PROVENANCE_CHECK_PROTECTED_FIELDS = frozenset({"kebechet_metadata"})
 
 
+CALLBACK_SECRET_NAME_TEMPLATE = "callback-{document_id}"
+
+
+def _add_item_to_callback_secret_entry(
+    document_id, callbackurl, auth_header: Optional[str] = None, client_data: Optional[dict] = None
+):
+    value = json.dumps({"callbackurl": callbackurl, "Authorization": auth_header, "client_data": client_data})
+    entry_name = random.choices(string.ascii_letters, k=16)
+    body = [{"op": "add", "path": f"/data/{entry_name}", "value": value}]
+    k8.client.CoreV1Api.patch_namespaced_secret(
```

Comment:
should we try to use `common` repo , so that if this kind action is needed somewhere else, we can use it ? 
we would have to create a patch_secret using oc
Example: https://github.com/thoth-station/common/blob/5645625b9b7411a78187cd22b5a0410913e2145d/thoth/common/openshift.py#L1114

Reply:
I don't think adding it to common is a good idea. I think creating a function in common will not remove any of the complexity from the call and it will just add another layer of abstraction.